packages/core/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -137,19 +141,25 @@ class AddressDetails extends Component {
                                                                     <DesktopWrapper>
                                                                         <Autocomplete
                                                                             {...field}
+                                                                            {...(this.state.addres_state_to_display && {
+                                                                              value: this.state.addres_state_to_display,
+                                                                            })}
                                                                             data-lpignore='true'
                                                                             autoComplete='new-password' // prevent chrome autocomplete
                                                                             dropdown_offset='3.2rem'
                                                                             type='text'
                                                                             label={localize('State/Province')}
                                                                             list_items={this.props.states_list}
-                                                                            onItemSelection={({ value, text }) =>
-                                                                                setFieldValue(
-                                                                                    'address_state',
-                                                                                    value ? text : '',
-                                                                                    true
-                                                                                )
-                                                                            }
+                                                                            onItemSelection={({ value, text }) => {
+                                                                              setFieldValue(
+                                                                                  'address_state',
+                                                                                  value ? text : '',
+                                                                                  true
+                                                                              );
+                                                                              this.setState({
+                                                                                  addres_state_to_display: '',
```

Comment:
Why are we resetting the address_state_to_display here?

Reply:
Just to be clear about the state.